web-app/src/components/BookingModal/index.js

```diff
@@ -4,8 +4,18 @@ import ModalStatusBanner from './ModalStatusBanner';
 import { PropTypes as PT } from 'prop-types';
 import BookingModalForm from './BookingModalForm';
 import { Modal } from '../common';
+
 import { StyleContainer, FormContainer } from './styled';
 
+const rejectionReason = booking => {
+
+  if (booking.messages && booking.messages.length > 0) {
+    return booking.messages[0].message;
+  }
+  return undefined;
```

Comment:
I think you'd possibly want to throw something nicer here than undefined.

Reply:
The returned undefined is checked within the modal to dyamically render the rejection message or not depending if there is one